_posts/2020-12-19-fixing-apple-smart-keyboard-with-copper-tape.md

```diff
@@ -0,0 +1,74 @@
+
+I did some research on the Internet and found out that many people had faced the same issue with Apple Smart Keyboard. I also found YouTube videos [1], [2] explaining how we can fix it with copper tapes or soldering.
+
+## Cause
+
+Apple Smart Keyboard uses conductive tapes to connect the Smart Connectors with keyboard electronics. Allegedly, there are two common issues with this connection. Either the connection between Smart Connectors and conductive tape is worn out(as it happened in my case) or the conductive tape itself is worn out(as shown in the YouTube video). In both cases, the keyboard wouldn't get recognized by the iPad.
+
+![Smart Keyboard Connector Pins](/img/smart-keyboard-connector-pins.jpg)
+
+![Smart Keyboard Schematic](/img/smart-keyboard-schematic.jpg)
+
+## Fix
+
+The fix is as simple as reestablishing the connection between Smart Connector and conductive tape or between the disconnected areas of the conductive tape.
+
+The YouTube videos tried best to hide the patchwork within the outer layer of the keyboard. The videos soldered a thin wired between the Smart Connector and the conductive to hide the patchwork. Despite having soldering kit and having done soldering multiple times, I am not good at soldering tiny components. I was concerned about damaging the Smart Connector pins or the keyboard shell. At the same time, I wasn't too much concerned aesthetic. So, I decided to avoid soldering and keep all the patchwork on the outer layer of the keyboard itself.
+
+![Smart Keyboard Fix](/img/smart-keyboard-final-fix.jpg)<!-- .element height="50%" width="50%" -->
+
+![Smart Keyboard Schematic Fix](/img/smart-keyboard-schematic-fix.jpg)
+
+### Step 1 - Peeling off the outer layer
+
+This was easier than I anticipated. I don't know whether it was because of the one year use or use of weak glue in manufacturing, the outer layer of the keyboard came off neatly without much of struggle. The trick is finding the starting point where the outer layer is already detached from the inner frame.
+
+### Step 2 - Identifying disconnection
+
+Once the outer layer is peeled off, the conductive tapes are readily visible and accessible. Use a multimeter or continuity tester to check the connection between each Smart Connector pins and the conductive tapes. There should be one or more disconnections. If you don't find a disconnection, then it must be a different problem why your keyboard is not getting recognized by the iPad, and the following steps won't help.
```

Comment:
> then it must be a different problem why your keyboard is not getting recognized by the iPad

Here: Can we reword to say something like `then you might have a different problem from me and these steps wont work?`